lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :network_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @network_separator = '/'
+      @external_copy_base = '/programdata'
```

Comment:
That doesn't look right... I just created a Bitvise host and tested manually with SCP (outside of Beaker), and what I suggested works fine:

```
[~] curl -d --url vmpooler.delivery.puppetlabs.net/vm/win-2012r2-wmf5-bv-x86_64                         13:18:53
{
  "ok": true,
  "win-2012r2-wmf5-bv-x86_64": {
    "hostname": "wu7b6kzeaaxxsmf",
    "ok": true
  },
  "domain": "delivery.puppetlabs.net"
}%                                                                                                                [~] scp ~/dotfiles.sublime-project "Administrator@wu7b6kzeaaxxsmf.delivery.puppetlabs.net:/PROGRA~3/dotfiles.sublime-project"
The authenticity of host 'wu7b6kzeaaxxsmf.delivery.puppetlabs.net (10.32.124.33)' can't be established.
ECDSA key fingerprint is SHA256:EKu2q+hGnVuwl36rb/2C0JlF2p3f90CMOTmwJkt4wSs.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added 'wu7b6kzeaaxxsmf.delivery.puppetlabs.net,10.32.124.33' (ECDSA) to the list of known hosts.
dotfiles.sublime-project                                                        100%   59     0.1KB/s   00:00
```

While I was testing this, I also noted that 8.3 compatibility is not actually necessary for Bitvise SCP either:

```
[~] scp ~/dotfiles.sublime-project "Administrator@wu7b6kzeaaxxsmf.delivery.puppetlabs.net:/Program\ Files/dotfiles.sublime-project"
dotfiles.sublime-project                                                        100%   59     0.1KB/s   00:00
```

For reference, under Cygwin SCP, the following works (with slashes reversed direction):

```
[~] curl -d --url vmpooler.delivery.puppetlabs.net/vm/win-2012r2-wmf5-x86_64                            13:24:21
{
  "ok": true,
  "win-2012r2-wmf5-x86_64": {
    "hostname": "nd9gn3322qzrt78",
    "ok": true
  },
  "domain": "delivery.puppetlabs.net"
}%

[~] scp ~/dotfiles.sublime-project "Administrator@nd9gn3322qzrt78.delivery.puppetlabs.net:\\PROGRA~3\\dotfiles.sublime-project"
dotfiles.sublime-project                                                        100%   59     0.1KB/s   00:00


[~] scp ~/dotfiles.sublime-project "Administrator@nd9gn3322qzrt78.delivery.puppetlabs.net:\\Program\ Files\\dotfiles.sublime-project"
dotfiles.sublime-project                                                        100%   59     0.1KB/s   00:00
```